### ISSUE TITLE
Pull down history white corners

### ISSUE THREAD

**Issue by MKrabs** [Darkempire78/OpenCalc#74]
![Screenshot_20221107_174119_Calculator](https://user-images.githubusercontent.com/71791731/200370267-100213d8-5846-4ad1-af44-32e7c371ed37.jpg)

**Comment by nayeemdv** [comment]
Need some clearification 👍 

As I understand this white corner should be removed.
This panel should be square until it returns to its default position

**Comment by Darkempire78** [comment]
@nayeemdv I don't think it would be aesthetically beautiful if the panel becomes square when you shoot it (except maybe with a nice animation?), the idea is that the white corners are transparent so that it is perfectly rounded, but I don't even know if it's possible, that's why I made a kind of illusion by using the same color as the background so that we believe that it is well rounded

**Comment by MKrabs** [comment]
To clarify:

1. Yeah the corners should be transparent
2. A cool animation could be to lower the radius of the corners from (idk) 30 to 0 while scrolling from pos 0 to max.

**Comment by inson1** [comment]
I think this is the biggest GUI issue.
